### DISCUSSION TITLE
How to define reverseOf pointing to a nested field

### DISCUSSION THREAD

**Discussion by ezzle** [apostrophecms/apostrophe#4025] [Q&A]
Hi.

From 'essential/piece/frametype/class'

`"eas_own_slot_value": {
				"label": "Essential OwnSlotValues",
				"type": "array",
				"required": true,
				"fields": {
					"add": {
						"_slot_reference": {
							"label": "Slot Reference",
							"type": "relationship",
							withType: 'essential/piece/frametype/slot',
							"required": true
						}, ...`

How to define the **reverseOf** in 'essential/piece/frametype/slot' ? Use of **dot** notation like '**eas_own_slot_value._slot_reference**' ?

`"_eas_own_slot_value_slot_references": {
				label: 'Essential OwnSlotValuesSlotReferences Domain',
				required: true,
				type: 'relationshipReverse',
				withType: 'essential/piece/frametype/class',
				reverseOf: 'eas_own_slot_value._slot_reference'
			}`

Best regards

Tshitshi

**Comment by boutell** [comment]
Hmmm, this looks like Apostrophe but also not. "essential/piece/frametype/class" is not any pattern that I recognize for apostrophe module names. Possibly you've chosen to use slashes in module names, outside of our support for one level of npm namespacing. Possibly this works, but it hasn't been tested FYI.

OK, having adjusted my eyeballs for that part, I see what you're asking. You cannot "reverse" a relationship that lives in an array property of document type. You can only reverse a relationship that lives in a top-level property of another document type.
